include/emitc/emitc_core_ops.h

```diff
@@ -252,6 +252,70 @@ inline Dest reshape(Src x) {
   return z;
 }
 
+// PadOp
+// TODO support negative edge padding
```

Comment:
```suggestion
// TODO: Add support for negative edge padding
```
It started as a nit (missing `:`) and ended in a rewritten comment :see_no_evil: